apis/coursebatchmanapi/coursebatchmanapi.yaml

```diff
@@ -56,7 +56,843 @@ paths:
         required: true
         type: string
         description: >-
-            Type of the request data. Set as application/json.
+            The Content Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'The time stamp at which the **create batch** request was sent. '
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'It is a Unique ID that identifies the request incase the same API is executed multiple times.'

```

Comment:
It is a Unique ID that identifies the request in case the same API is executed multiple times.'